tests/task_queue_consumer/task_queue_consumer_test.py

```diff
@@ -188,22 +205,105 @@ def test_sanitises_input_dodgy_input():
 # covered
 @pytest.mark.unit
 @serialise_mocks()
-@resetting_mocks(task_queue_consumer.ecs_client, task_queue_consumer.ssm_client)
+@resetting_mocks(task_queue_consumer.ecs_client, task_queue_consumer.ssm_client.get_parameters)
 def test_sanitises_input_dodgy_input_url():
-    task_queue_consumer.ssm_client.get_parameters.return_value = ssm_return_vals(False)
+    task_queue_consumer.ssm_client.get_parameters.return_value = ssm_return_vals(
+        False)
     with pytest.raises(ValueError):
         task_queue_consumer.submit_scan_task(
-            {"Records": [{"body": "http://foo.bar?hello=foo; rm -rf", "messageId": "13"}]},
+            {"Records": [
+                {"body": "http://foo.bar?hello=foo; rm -rf", "messageId": "13"}]},
             mock.MagicMock())
 
 
 @pytest.mark.unit
 @serialise_mocks()
-@resetting_mocks(task_queue_consumer.ecs_client, task_queue_consumer.ssm_client)
+@resetting_mocks(task_queue_consumer.ecs_client.run_task, task_queue_consumer.ssm_client.get_parameters)
 def test_raises_when_ecs_failures_present():
-    task_queue_consumer.ssm_client.get_parameters.return_value = ssm_return_vals(False)
+    task_queue_consumer.ssm_client.get_parameters.return_value = ssm_return_vals(
+        False)
     task_queue_consumer.ecs_client.run_task.return_value = {'failures': [
         {'arn': 'arn::some:::arn', 'reason': 'failed miserably'}
     ]}
     with pytest.raises(RuntimeError, match=r"\{\"arn\": \"arn::some:::arn\", \"reason\": \"failed miserably\"\}"):
-        task_queue_consumer.submit_scan_task({"Records": [{"body": "some.host", "messageId": "13"}]}, mock.MagicMock())
+        task_queue_consumer.submit_scan_task(
+            {"Records": [{"body": "some.host", "messageId": "13"}]}, mock.MagicMock())
+
+
+@pytest.mark.unit
+@serialise_mocks()
+@resetting_mocks(task_queue_consumer.ecs_client, task_queue_consumer.ssm_client.get_parameters)
+def test_sanitises_input_ip4_json():
```

Comment:
CAn we change the names of these 4 tests please? They aren't really testing the sanitising, they are testing alternative input syntax and grouping